src/api/admin/admin.ts

```diff
@@ -5,6 +5,10 @@ import AdminController from './controller/admin';
 const router = express.Router();
 const adminController = new AdminController();
 
+router.get('/', adminController.handleGetDraftWishcards);
```

Comment:
please do not use plain `/` routes anymore, choose a descriptive name what the route does